app/views/books/_book_form.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_with model: @book do |form| %>
+  <%= form.label :title, 'Title' %>
+  <%= form.text_field :title %>
+  <%= form.label :author, 'Author' %>
+  <%= form.text_field :author %>
+  <%= form.label :isbn, 'ISBN' %>
+  <%= form.text_field :isbn %>
+  <%= form.submit %>
+<% end %>
```

Comment:
This will let empty fields be submitted. Was that the intended behaviour?

Reply:
Why would it @liliannaszabo ?

Reply:
I don't have validations on the model and this form doesn't enforce any client-side validations